app/server/lib/BrowserSession.ts

```diff
@@ -70,10 +70,17 @@ export interface SessionObj {
                           // to edit other people's contributions).
 
   oidc?: {
```

Comment:
Can we define the `oidc` part of the session as a named interface instead? Then in `Protections.ts` we can use the name of the interface (rather than `Exclude<Sessionbj['oidc'], undefined>`) which has the advantage that when someone wants to find where this type is used in the codebase, they could just search for the name.